src/main/java/fi/metatavu/soteapi/rss/news/NewsListJob.java

```diff
@@ -157,12 +158,7 @@ private NewsUpdateModel createModel(SyndEntry syndEntry) {
       }
     }
     
-    NewsUpdateModel result = new NewsUpdateModel();
-    result.setCategorySlug(categorySlug);
-    result.setContent(content);
-    result.setSlug(slug);
-    result.setTitle(title);
-    result.setOriginId(originId);
+    NewsUpdateModel result = new NewsUpdateModel(title, content, slug, originId, categorySlug, orderIndex);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)